internal/sql/repository/security/ImageScanDeployInfoRepository.go

```diff
@@ -185,7 +185,8 @@ func (impl ImageScanDeployInfoRepositoryImpl) scanListQueryWithoutObject(request
 	}
 	query = query + " INNER JOIN environment env on env.id=info.env_id"
 	query = query + " INNER JOIN cluster clus on clus.id=env.cluster_id"
-	query = query + " WHERE info.scan_object_meta_id > 0 and env.active=true"
+	query = query + " INNER JOIN app ap on ap.id = info.scan_object_meta_id"
```

Comment:
scan_object_meta_id is not exactly the appId, it contains other ids based on object type